src/components/PeopleFilters.tsx

```diff
@@ -1,18 +1,51 @@
-export const PeopleFilters = () => {
+import React from 'react';
+import cn from 'classnames';
+
+interface Props {
+  sex: string;
+  query: string;
+  centuries: string[];
+  onReset: (param: string) => void;
+  onSexChange: (sexFilter: string) => void;
+  setCenturies: (century: string) => void;
+  onQueryChange: (event: React.ChangeEvent<HTMLInputElement>) => void;
+}
+
+export const PeopleFilters: React.FC<Props> = ({
+  sex,
+  query,
+  centuries,
+  onSexChange,
+  onReset,
+  onQueryChange,
+  setCenturies,
+}) => {
+  const handleResetAll = () => {
+    onReset('sex');
+    onReset('query');
+    onReset('centuries');
+  };
+
+  const centuriesArray = ['16', '17', '18', '19', '20'];
+
   return (
     <nav className="panel">
       <p className="panel-heading">Filters</p>
 
       <p className="panel-tabs" data-cy="SexFilter">
-        <a className="is-active" href="#/people">
-          All
-        </a>
-        <a className="" href="#/people?sex=m">
-          Male
-        </a>
-        <a className="" href="#/people?sex=f">
-          Female
-        </a>
+        {[
+          { param: '', title: 'All' },
+          { param: 'm', title: 'Male' },
+          { param: 'f', title: 'Female' },
+        ].map(filter => (
```

Comment:
it would've been better if you put that array into constant instead of what you did

```tsx
const filterOptions = [
          { param: '', title: 'All' },
          { param: 'm', title: 'Male' },
          { param: 'f', title: 'Female' },
        ]